clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -45,6 +46,14 @@
 using namespace cir;
 using namespace llvm;
 
+// Actual lowering
```

Comment:
You're missing the beginning of the function, and this should be sunk down to be located near the other floating point operations,